lib/matplotlib/quiver.py

```diff
@@ -88,13 +88,14 @@
     Method for determining the angle of the arrows.
 
     - 'uv': Arrow direction in screen coordinates. Use this if the arrows
-      symbolize a quantity that is not based on *X*, *Y* data coordinates.
+      symbolize normalized x-y components.
```

Comment:
I don't think this is clear. First, what does "normalized" mean here? It's a term with multiple connotations. Second, mentioning "x-y" is a red herring, because it's close to *X* and *Y*, which we specifically don't want to associate here.

If don't find the original wording too bad. At it's core there is a negation: "all data that are not related to the data coordinates". You could try to work around the "not" by

> that is unrelated to *X*, *Y* data coordinates.

but 🤷

Reply:
Because of my engineering background, I thought "normalized" was a standard word. My bad. Maybe something like unitary?